includes/entities/class-fs-plugin.php

```diff
@@ -84,6 +88,17 @@ function is_addon() {
 			return isset( $this->parent_plugin_id ) && is_numeric( $this->parent_plugin_id );
 		}
 
+		/**
+		 * Returns "true" if the type is not "theme".
+		 *
+		 * @author Leo Fajardo (leorw)
+		 *
+		 * @return bool
+		 */
+		function is_plugin() {
+			return ( 'theme' !== $this->type );
```

Comment:
@fajardoleo for backward compatibility I would use:
`return ( ! isset($this->type) || 'theme' !== $this->type );`